capstone/cap/models.py

```diff
@@ -251,22 +251,36 @@ class CaseMetadata(models.Model):
     slug = models.SlugField(unique=True, max_length=255)
     first_page = models.IntegerField(null=True, blank=True)
     last_page = models.IntegerField(null=True, blank=True)
-    jurisdiction = models.ForeignKey('Jurisdiction', null=True, related_name='%(class)s_jurisdiction', on_delete=models.SET_NULL)
-    citation = models.CharField(max_length=255, blank=True)
+    jurisdiction_xml = models.CharField(max_length=255, blank=True)
```

Comment:
I think that we should convert the xml to json, and then have this be a json or jsonb field. If that doesn't work for some reason, I think we should at least use the Postgres XML field type so we can query the enclosed data directly rather than having to grab the string and parse it locally. Thoughts?